Build/Common.Build.props

```diff
@@ -90,6 +90,13 @@
         BYTECODE_TESTING=1
       </PreprocessorDefinitions>
       <PreprocessorDefinitions Condition="'$(BuildWabt)'=='true'">%(PreprocessorDefinitions);CAN_BUILD_WABT=1</PreprocessorDefinitions>
+
+      <PreprocessorDefinitions Condition="'$(ChakraVersionBuildNumber)'!=''">%(PreprocessorDefinitions);CHAKRA_VERSION_BUILD_NUMBER=$(ChakraVersionBuildNumber)</PreprocessorDefinitions>
```

Comment:
This change makes these #defines available to the C++ compiler in addition to the ResourceCompiler. With this change, we can just include the version definition files & they'll have the same values in c++ that they do during resource compilation.